lib/mayaUsd/resources/ae/usd-shared-components/src/python/usdSharedComponents/common/filteredStringListView.py

```diff
@@ -2,42 +2,44 @@
 from .theme import Theme
 from .host import Host
 from .filteredStringListModel import FilteredStringListModel
+from ..data.stringListData import StringListData
 
 try:
     from PySide6.QtCore import (  # type: ignore
-        QItemSelection,
-        QItemSelectionModel,
+        QEvent,
         QModelIndex,
+        QObject,
         QPersistentModelIndex,
         QSize,
         QStringListModel,
         Qt,
         Signal,
     )
-    from PySide6.QtGui import QPalette, QPainter, QPaintEvent  # type: ignore
+    from PySide6.QtGui import QPainter, QPaintEvent, QFont  # type: ignore
     from PySide6.QtWidgets import (  # type: ignore
-        QFrame,
-        QLabel,
         QListView,
         QStyledItemDelegate,
         QStyleOptionViewItem
     )
 except:
     from PySide2.QtCore import (  # type: ignore
-        QItemSelection,
-        QItemSelectionModel,
+        QEvent,
         QModelIndex,
+        QObject,
         QPersistentModelIndex,
         QSize,
         QStringListModel,
         Qt,
         Signal,
     )
-    from PySide2.QtGui import QPalette, QPainter, QPaintEvent  # type: ignore
-    from PySide2.QtWidgets import QFrame, QLabel, QListView, QStyledItemDelegate, QStyleOptionViewItem  # type: ignore
+    from PySide2.QtGui import QPainter, QPaintEvent, QFont  # type: ignore
+    from PySide2.QtWidgets import QListView, QStyledItemDelegate, QStyleOptionViewItem  # type: ignore
 
 
 kNoObjectFoundLabel = "No objects found"
+kDragObjectsHereLabel = "Drag objects here"
+kPickObjectsLabel = "Click '+' to add"
+kDragOrPickObjectsLabel = "Drag objects here or click '+' to add"
```

Comment:
doesn't python normally use UPPER_CASE for constants? 
https://peps.python.org/pep-0008/#constants

Reply:
In the MayaUSD repo, the coding standard is to have constant be in the form I used, that's why I followed it in this case, as other Maya dev would have chimed in to make the constant follow the coding standard of the project. I can change them, but whatever standard we follow will clash with either DCC.